src/main/java/aurora/parser/Parser.java

```diff
@@ -0,0 +1,171 @@
+package aurora.parser;
+
+import aurora.command.ByeCommand;
```

Comment:
I think the checkstyle document they use have a different order of imports.
>The order rule "STATIC###STANDARD_JAVA_PACKAGE###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE" consists of:
      1. STATIC: static imports
      2. STANDARD_JAVA_PACKAGE: standard java/javax imports
      3. SPECIAL_IMPORTS: defined as org imports
      4. THIRD_PARTY_PACKAGE: defined as com imports